data/templates/install-ubuntu/ubuntu-preseed

```diff
@@ -57,6 +63,11 @@ d-i time/zone string US/Pacific
 d-i clock-setup/ntp boolean true
 
 ### Partitioning
+d-i partman/early_command string \
+<%_ if( typeof progressMilestones !== 'undefined' && progressMilestones.startPartitionUri ) { _%>
+wget 'http://<%=server%>:<%=port%><%-progressMilestones.startPartitionUri%>' || true;
```

Comment:
If the examine the while `ubuntu-preseed`, it keeps the style that doesn't indent the command. So I am just copying the previous style as I don't want to refactor all previous code.

if the original file indents the command, then I choose to indent my command as well.